botfront/imports/ui/components/common/style.less

```diff
@@ -0,0 +1,36 @@
+.ui.vertical.menu .item>#selected {
```

Comment:
try and leverage .less possibilities, for instance you could have put .ui.vertical.menu only once and then nest other declarations inside. Come and see me if you don't understand what I mean

Reply:
I will come and see you

Reply:
Done